test/httpwstest.cpp

```diff
@@ -233,9 +211,6 @@ void HTTPWSTest::testReloadWhileDisconnecting()
         TST_LOG("Loading again.");
         socket = loadDocAndGetSession(_socketPoll, _uri, documentURL, testname);
 
-        // Should have no new instances.
-        LOK_ASSERT_EQUAL(kitcount, countCoolKitProcesses(kitcount));
```

Comment:
Surely there is a useful check here we lost ?

Reply:
This test fails and is not running. It passes if you wait between disconnect and reconnect, but since it's called testReloadWhileDisconnecting I think that specifically should not be done. I'll add a comment here explaining that much. When it gets fixed and reenabled we can add whatever waits or asserts we need.

Reply:
Fine - it's not included in the set currently.